docs/how-to-guides/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -172,7 +174,11 @@ When defining a child block, use the `parent` block setting to define which bloc
 }
 ```
 
-Another example is using the `ancestors` block setting to define a block that must be present as an ancestor, but it doesn't need to be the direct parent (like with `parent`). This prevents the block from showing in the inserter if the ancestor is not in the tree, but other blocks can be added in between, like a Columns or Group block. See [Comment Author Name code for reference](https://github.com/WordPress/gutenberg/tree/HEAD/packages/block-library/src/comment-author-name).
+### Defining Ancestor Block Context
+
+Another example is using the `ancestor` block setting. The Comment Author Name block utilizes the `ancestor` setting to assign the Comment Template block to allow it to be a descendant of the Comment Template block while refraining from explicitly assigning it a direct child hierarchy. 
+
+This context allows the Comment Author Name block to be anywhere in the hierarchical tree, and not _just_ a direct child of the parent Comment Template block, while still limiting its availability within the block inserter to only be visible an an option to insert if the Comment Template block is available.
```

Comment:
I find this change a little more unclear than the original description. 

In the `parent` section above it starts with a generalized explanation first and then dives into an example. This section here is now missing that general section entirely. (It had that in the prior documentation). I would love to see a more general explanation before it dives into the example.